mongodbatlas/resource_mongodbatlas_project.go

```diff
@@ -135,10 +140,18 @@ func resourceMongoDBAtlasProjectCreate(ctx context.Context, d *schema.ResourceDa
 	// Get client connection.
 	conn := meta.(*MongoDBClient).Atlas
 
+	var regionUsageRestrictions string
+
+	mongodbgovCloud := pointy.Bool(d.Get("is_mongodbgov_cloud").(bool))
+	if *mongodbgovCloud {
+		regionUsageRestrictions = "GOV_REGIONS_ONLY"
```

Comment:
Wait - actually @martinstibbe I think two different things got mixed up here and I didn't realize it till I hit submit on my review. There's having all the API requests coming from the provider go to gov cloud - that's one thing. That's why I suggested to have the gov part in the actual provider settings. But this is mixing up setting a project to only use gov cloud regions as setting the whole provider to use the gov cloud url. This should actually, imho, be two different settings. The provider should be able to be pointed to the different baseURL. Then one when creating a project should be able to set "regionUsageRestrictions" : "GOV_REGIONS_ONLY" As the docs say "By default, the AtlasGov API designates new projects for standard region usage. To use the API to create a project for government regions, include the following in your POST request data" Happy to chat about this to clear it up and we can confirm my understanding with Salman (PM for govcloud).